WordPress/Classes/Models/Role.swift

```diff
@@ -12,6 +12,15 @@ extension Role {
     func toUnmanaged() -> RemoteRole {
         return RemoteRole(slug: slug, name: name)
     }
+
+    /// Returns a role from Core Data with the given slug.
```

Comment:
Nitpick: This comment might be misleading. It mentions "the given slug" but ignores the `withBlogID` input parameter, which is used in the implementation.

I'd suggest removing the comment entirely and rely solely on an already clear function signature. What do you think?

```suggestion
```